### PR TITLE
Adding fastify-jaeger-middleware plugin to the community ecosystem

### PR DESCRIPTION
**Pull request by perljusback**
Greetings, 

We've created a middleware for fastify with jaegertracing to support our microservices architecture, we looked around for something similiar but didn't find anything so we thought we'd share it here.

Best Regards
Per

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [ ] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Ecosystem.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ Plugins maintained by the fastify team are listed under [Core](#core) while plug
 - [`fastify-hemera`](https://github.com/hemerajs/fastify-hemera) Fastify Hemera plugin, for writing reliable & fault-tolerant microservices with [nats.io](https://nats.io/).
 - [`fastify-jwt-authz`](https://github.com/Ethan-Arrowood/fastify-jwt-authz) JWT user scope verifier.
 - [`fastify-jwt-webapp`](https://github.com/charlesread/fastify-jwt-webapp) JWT authentication for fastify-based web apps.
+- [`fastify-jaeger-middleware`](https://github.com/3rdi-ai/fastify-jaeger-middleware) Jaeger Tracing Middleware for fastify.
```
Comment (mcollina) [inline docs/Ecosystem.md]: I think this should go before `fastify-jwt`. THe list is in alphabetical order.

**Comment by StarpTech** [comment]
@mcollina did you review it? I propose to add tests before we will list it as an official module.

**Comment by StarpTech** [comment]
This is what I mean. Testing fastify plugin interface and check that correct span, tags are logged for error and success case.

**Comment by cemremengu** [comment]
Name of the repo can be confusing I think `fastify-jaeger-middleware` since it is a plugin. Maybe rename repo to `fastify-jaeger-plugin` if possible ?

**Comment by perljusback** [comment]
@mcollina it's in our pipeline to add tests to our plugins, I'll see if i can up prioritize it.

@cemremengu we can change it if you guys want, the reasoning internaly (coming from express) was that it acts as a middleware.

**Comment by cemremengu** [comment]
@perljusback Looks like this plugin is deprecated https://www.npmjs.com/package/@3rdi/fastify-jaeger-middleware

**Comment by delvedor** [comment]
Since the plugin has been marked as deprecated I'm closing this pr.

**Comment by arwineap** [comment]
@delvedor To be clear, I think the old module was deprecated in favor of the renamed one; as suggested by @mcollina 

https://www.npmjs.com/package/@3rdi/fastify-jaeger-middleware
vs
https://www.npmjs.com/package/fastify-jaeger

**Comment by perljusback** [comment]
@arwineap We deprecated it for adding of tests and for a rename, with that said https://www.npmjs.com/package/fastify-jaeger is not created by us.

**Pull request closed without merging**